setup.py

```diff
@@ -62,32 +55,25 @@
     setup_requires = ["setuptools_scm"]
 
     interactive_requirements = [
```

Comment:
make interactive_requirements in all repos, should include: ipython, ipywidgest, jupyter, matplotlib

lump ipython, ipywidgets, and jupyter together in `jupyter` key, e.g.

interactive_requirements = [
    vbu[jupyter],
    vbu[matplotlib],
]

Reply:
Ask RT what they always have to explicity import b/c it doesn't with the model repo for doing interactive stuff